docs/proposals/proposal-002-run.md

```diff
@@ -0,0 +1,293 @@
+
+The benchmark workflows will be stored in the same JSON file as the other parameters for CNCF projects as defined in [Proposal 1](./proposal-001-trigger-and-deploy.md). It can be added as an additional input.
+
+```yaml
+# .github/workflows/benchmark-pipeline.yaml
+jobs:
+  # first, must authenticate to the Kubernetes cluster
+  # this is a benchmark job
+  benchmark-job:
+    # benchmark job calls on benchmark workflow
+    uses: ${{ inputs.benchmark_path }} # refers to benchmark workflow path
+```
+
+This will fetch the workflow using the `jobs.<job_id>.uses` syntax defined [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iduses).
+
+Below are two use cases: the benchmark workflow may be defined in the Green Reviews repository or in a separate repository.
+
+#### Use Case 1: A GitHub Action job using a workflow defined in the _same_ repository (preferred)
+
+If the benchmark workflow is located in the Green Reviews repository, `benchmark_path` would refer to, for example, `cncf-tags/green-reviews-tooling/.github/workflows/falco-benchmark-workflow.yml@v1`.
+
+In terms of the directory structure, in the `green-reviews-tooling` repository, we could create a subfolder such as `./github/workflows/benchmark-workflows` to contain the benchmark workflows.
+
+#### Use Case 2: A GitHub Action job using a workflow defined in a _different_ repository
+
+We want to accommodate different methods of setting up the tests depending on the CNCF project. Given this, the benchmark workflow containing the benchmark job could be defined in a different repository. In this case, the `benchmark_path` would be, for example, `falcosecurity/cncf-green-review-testing/.github/workflows/workflow.yml@v1`.
+
+![Pipeline run](diagrams/pipeline-run.png "An example pipeline run")
+
+### Benchmark jobs
+
+The benchmark workflow which contains the benchmark jobs and their test steps may look like the following:
+
+```yaml
+# .github/workflows/tests/falco-benchmark-workflow.yaml
+jobs:
+  stress-ng-test:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          # the action to take here depends on the Functional Unit of the CNCF project. wait for amount of time, for resources
+          kubectl apply -f https://raw.githubusercontent.com/falcosecurity/cncf-green-review-testing/main/kustomize/falco-driver/ebpf/stress-ng.yaml
+          wait 15m
+      - delete: |
+         kubectl delete -f https://raw.githubusercontent.com/falcosecurity/cncf-green-review-testing/main/kustomize/falco-driver/ebpf/stress-ng.yaml  # other Falco tests:
+  # - redis-test e.g. https://github.com/falcosecurity/cncf-green-review-testing/blob/main/kustomize/falco-driver/ebpf/redis.yaml
```

Comment:
Maybe it is more clear to put this comment below the run and before the delete? See suggestion above